mlflow/tracking/_tracking_service/client.py

```diff
@@ -311,7 +344,7 @@ def set_experiment_tag(self, experiment_id, key, value):
         tag = ExperimentTag(key, str(value))
         self.store.set_experiment_tag(experiment_id, tag)
 
-    def set_tag(self, run_id, key, value):
+    def set_tag(self, run_id, key, value, synchronous: bool = True) -> RunOperations:
```

Comment:
same here: remove type annotation.

Reply:
I have added annotation for all the parameters for the APIs I touched in this PR. Let me know if you not to do this at all and should revert back.

Reply:
Removed annotations from this file as per suggestion.